package.json

```diff
@@ -10,7 +10,7 @@
     "docker:major": "yarn version --major",
     "docker:login": "docker login ghcr.io",
     "docker:tag": "docker tag $npm_package_name:latest ghcr.io/greenroom-robotics/$npm_package_name:$npm_package_version",
-    "docker:push": "docker push ghcr.io/greenroom-robotics/$npm_package_name:$npm_package_version",
+    "docker:push": "docker push --all-tags ghcr.io/greenroom-robotics/$npm_package_name:$npm_package_version",
```

Comment:
Do we need to tag latest as ghcr.io/greenroom-robotics/$npm_package_name: latest?

Reply:
Ah yep. Will fix it.